src/data_manager/data_file_manager.py

```diff
@@ -128,61 +147,65 @@ def process_config(self):
         # Create our DataTypeConfig (which in turn create our SubTypeConfig) objects.
         self.dispatch_to_object()
 
-    def _search_submission_data(self, dataType, subEntry):
-
-            subType = subEntry
-
-            try:
-                returned_dict = next(item for item in self.submission_system_data['snapShot']['dataFiles']
-                                     if item['dataType'].get('name') == dataType and item['dataSubType'].get('name') == subType)
-
-            except StopIteration:
-                logger.debug('dataType: %s subType: %s not found in submission system data.' % (dataType, subType))
-                logger.debug('Creating entry with \'None\' path and extracted path.')
-                returned_dict = {
-                    'dataType': dataType,
-                    'subType': subType,
-                    'path': None,
-                    'tempExtractedFile': None
-                }
-
-            return returned_dict
+    def _search_submission_data(self, data_type, sub_type):
+
+        try:
+            returned_dict = next(item for item in self.submission_system_data['snapShot'] \
```

Comment:
if we break up this into multiple lines, can we put the multi-line bits at the same indentation level perhaps?